chemprop/cli/train.py

```diff
@@ -458,6 +458,12 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         default=0,
         help="Seed for PyTorch randomness (e.g., random initial weights).",
     )
+    parser.add_argument(
+        "--patience",
+        type=int,
+        default=None,
+        help="Number of epochs to wait for improvement before early stopping.",
+    )
```

Comment:
Note: this can stay because we've discussed this already, but stuff like this should be a separate PR in the future (there really is no minimum size!)

Reply:
Thanks for the suggestion! Will keep this change here and open up new PRs in the future.